.github/instructions/assets.nuxt-ai.instructions.md

```
@@ -0,0 +1,337 @@
+## Examples
+
+<example>
+// Example of proper CSS handling with Nuxt assets
+export default `
+// assets/css/main.css
+@import './variables.css';
+@import './typography.css';
+@import './utilities.css';
+
+:root {
+/_ Use CSS variables for theming _/
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)